client/js/table-view.js

```diff
@@ -51,6 +53,12 @@ class TableView {
             this.currentCellLocation.row === row;
   }
 
+  renderTableFooter() {
+    for (let col = 0; col < this.model.numCols; col++) {
+      this.footerRowEl.appendChild(createTD(0));
```

Comment:
Why is this being initialized to 0?  The sum of 5 and -5 would be 0.  Or the sum of 0 and 0 is 0.  But if the cells for a given column are all empty, that column's sum is blank.